abs.yaml

```diff
@@ -0,0 +1,11 @@
+# disable tests on initial build due to circular dependency with pytorch cuda.
+build_parameters:
+  - "--suppress-variables"
+  - "--skip-existing"
+  - "--error-overlinking"
+  - "--no-test"
+
+upload_without_merge: True
+upload_to_staging_channel_in_pr: True
+upload_channels: [ cuda12_def ]
```

Comment:
this only concerns uploading after merge: https://github.com/anaconda-distribution/rocket-platform/blob/94818797531dc0770f4455e3377a3d6f85621efc/cookbook/flows/config.yaml#L34